src/lib/components/FileUploader/FileUploaderToolbar.js

```diff
@@ -9,7 +9,7 @@ import { useFormikContext } from 'formik';
 import React from 'react';
 import { Checkbox, Grid, Icon, Label, List, Popup } from 'semantic-ui-react';
 import { humanReadableBytes } from './utils';
-import i18next from '../../i18next';
+import { i18next } from '../../i18next';
```

Comment:
can we import directly the `t` function, i think it would b more human-readable:
`import t from `../../i18next;`

Reply:
Eventually, if we do that the code to mark for translation the strings would become `t.t()` which seems kind of weird....I kept for the time being the

```javascript
import { i18next} from 'path/to/i18next'
i18next.t('Translate me!')
```